lib/omniauth/strategies/openid_connect.rb

```diff
@@ -35,7 +37,7 @@ class OpenIDConnect
       option :client_jwk_signing_key
       option :client_x509_signing_key
       option :scope, [:openid]
-      option :response_type, 'code'
+      option :response_type, 'code' # ['code', 'id_token', 'id_token token']
```

Comment:
same question here. Why do we need `'id_token token'` option?

Reply:
It's one of the valid response types for Implicit Grant.

Reply:
I know. But we don't have the full support of it according to the updated readme.